integration-tests/cypress/e2e/api/devices/devices.ts

```diff
@@ -1,8 +1,35 @@
 /// <reference path="../../../support/index.d.ts" />
 import { getTestName } from "@commands/names";
 import { makeAuthorizedRequest, v1ApiPath, getCreds } from "@commands/server";
+import { TestGetLocation } from "@commands/api/station";
```

Comment:
Roll back your changes to this file now that you've broken things out into a separate file.